### PR TITLE
user-vhost-warning

### PR DESCRIPTION
**Pull request by asaiacai**

### CODE REVIEW

**Comment by matthew-mcallister** [comment]
Can we not base this on the template @hexecute used last time? For reference
```
Dear {group}: 

The OCF staff noticed that your group's website [1] does not have the "Hosted
by OCF" banner required by the virtual hosting rules [2]. All student group
websites are required to display the "Hosted by OCF" banner as well as the 
student group disclaimer. Unfortunately, if you do not add both of them, we
will be forced to disable your website to comply with the university's
requirements. Please add both sometime within the next 3 weeks (by May 24) to
prevent disabling of your website. Thanks for your cooperation and good luck on
finals.

Your username is "{username}".

If you've forgotten how to login, instructions can be found here [3]. If you've
forgotten your password, you can reset it here [4].

Peter Wu
Deputy Manager

1. {website}
2. https://www.ocf.berkeley.edu/docs/services/vhost/#h3_including-the-ocf-banner
3. https://www.ocf.berkeley.edu/docs/services/shell/
4. https://www.ocf.berkeley.edu/account/password/
```
I do think the copy could use some work, but it's friendlier, more helpful, and more informative.

File: www/templates/user-vhost-warning [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Hi {name},
+
+We noticed that your virtual host {host.berkeley.edu} didn't have the required
+Hosted by OCF banner or University Disclaimer[2].  Please make sure you
+resolve this issue within 2 weeks time in order to avoid having it be disabled.
+
+[1] https://ocf.io/vhost#h3_including-the-ocf-banner
```
Comment (chriskuehl) [inline www/templates/user-vhost-warning]: if we're linking to this in an email, it should probably have a permanent anchor name

**Comment by matthew-mcallister** [comment]
Also, I think we should make a second template requiring the banner only for vhosts which are not run by student groups.

File: www/templates/user-vhost-warning [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+of them, we may be forced to temporarily disable your website to comply with the OCF's
+and university's requirements. Please add both sometime within the next 2 weeks to
+prevent disabling of your website.
+
+Your username is "{username}".
+
+If you've forgotten how to login, instructions can be found here [4]. If you've
+forgotten your password, you can reset it here [5].
+
+Feel free to reply all to this email if you have any questions or if we can help with making the change! Please include help@ocf.berkeley.edu if you would like a response.
```
Comment (kpengboy) [inline www/templates/user-vhost-warning]: If you use an RT ticket in the help queue, then `help@ocf.berkeley.edu` will already be the reply-to address. There wouldn't be a difference between reply and reply all either.

**Pull request merged**